aspnetcore/signalr/configuration.md

```diff
@@ -162,6 +162,31 @@ let connection = new signalR.HubConnectionBuilder()
     .build();
 ```
 
+::: moniker range=">= aspnetcore-3.0"
+
+Instead of a `LogLevel` value, you can also provide a `string` value representing a log level name (see below for a list). This is useful when configuring SignalR logging in environments where you don't have access to the `LogLevel` constants.
```

Comment:
Not sure "(see below for a list)" is needed. The article is short and you say the same thing in two sentences.